tests/hpxml_translator_test.rb

```diff
@@ -415,8 +415,8 @@ def _get_results(rundir, sim_time, workflow_time)
 
     sqlFile.close
 
-    assert_operator(compload_results["Heating - Residual"], :<, 0.2)
-    assert_operator(compload_results["Cooling - Residual"], :<, 0.2)
+    assert_operator(compload_results["Heating - Residual"], :<, 0.35)
+    assert_operator(compload_results["Cooling - Residual"], :<, 0.35)
```

Comment:
@yzhou601 😱

Reply:
It appears that the values are off by more than before because the residual value checks above are absolute values. In reality, the heating residuals are negative and cooling residuals are positive, with the total residuals very close to 0 for most files (but a few higher, max of 0.23 GJ). 

The "problem" is likely due to choosing the "wrong" season for an hour with no heating/cooling load. If a month in the Fall is assigned to heating and a month in the Spring is assigned to cooling, and these months see opposite trends to the living space temperature, their net sum loads will be non-zero. I'm going to try tweaking the heating/cooling seasons a bit but the issue will never be eliminated.

Reply:
True, we're assigning modes with definition for that situation, along with signs(+ for heating and - for cooling). If a wrong sign is picked, they won't be offset during runperiod. Just one thought: why can't we create another bin/mode for no heating/cooling load situation (eg. float_mode)? It can distinguish the general heat transfer happens all the time from load seen by hvac, and help go through the residual check. Though it might output a new set of component load variables with prefix "float_" or something.